src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java

```diff
@@ -45,22 +45,26 @@
  */
 public class TestSetupHooks {
 
-  @Before(order = 2, value = "@BQ_SINK_TEST")
-  public void setTempTargetBQTable() {
-    String bqTargetTable = "TestSN_table" + RandomStringUtils.randomAlphanumeric(10);
-    PluginPropertyUtils.addPluginProp("bqTargetTable", bqTargetTable);
-    BeforeActions.scenario.write("BigQuery Target table name: " + bqTargetTable);
+  @Before(order = 1, value = "@CREATE_CUSTOM")
+  public static void createCustom() throws UnsupportedEncodingException, ParseException {
+    BeforeActions.scenario.write("Creating new Custom Object..");
+
+    JSONObject custom = new JSONObject(PluginPropertyUtils.pluginProp("CustomObjectData"));
```

Comment:
instead of naming it custom writ testData

Reply:
done